Automattic/wp-calypso
# Stats: Release horizontal bar chart modules

#### Proposed Changes

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/4044428/201447628-6b1339a9-a60a-4f04-a70f-5411d4791253.png">

Enables the new horizontal bar lists for all stats modules in the Traffic page.

#### Testing Instructions



* Open the Calypso live branch for this PR.
* Navigate to the Traffic Stats page and ensure all the Stats modules include horizontal bar chart backgrounds.

#### Pre-merge Checklist





Fixes #69044.
